src/content.rs

```diff
@@ -214,9 +174,13 @@ pub fn all_pages_load(dir: PathBuf, show_unpublished: bool) -> anyhow::Result<In
             }
         }
     }
+    let cache_expiration = match contains_unpublished {
+        true => earliest_unpublished.unwrap(),
+        false => DateTime::parse_from_rfc3339("2143-12-31T23:59:59+00:00")?.with_timezone(&Utc),
```

Comment:
What is the significance of this string?  If you want something far in the future, could you use `UTC_MAX` rather than parsing an oddly specific string?

If the specific string is required, consider pulling it out to a constant with a meaningful name.

Reply:
The string is just a magic string formed from editing UTC_MAX. The parser did not seem happy to parse UTC_MAX as it was of the form `yyyyyy-mm-ddTHH:MM::SS:mmmmmm+00:00`. I will make sure again to check I was not missing something and if not I will pull it out with a constant.

Reply:
You wouldn't parse MAX_UTC (sorry messed up the name) would you?  It's already a DateTime.  https://docs.rs/chrono/latest/chrono/struct.DateTime.html#associatedconstant.MAX_UTC

Reply:
This was just due to me parsing them incorrectly. Fixed it to use `MAX_UTC`.

Reply:
I was trying to convert it to string and back. My mistake was I was converting it to a different format.

Reply:
I might actually have to take a look at this again tomorrow as the date format used content is "rfc339". I have to double-check a couple of things tomorrow.